src/niscope/templates/session.py/fancy_fetch.py.mako

```diff
@@ -0,0 +1,76 @@
+<%page args="f, config, method_template"/>\
+<%
+    '''Forwards to _fetch_double()'''
+    import build.helper as helper
+
+    suffix = method_template['method_python_name_suffix']
+%>\
+    def ${f['python_name']}${suffix}(${helper.get_params_snippet(f, helper.ParameterUsageOptions.SESSION_METHOD_DECLARATION)}):
+        '''${f['python_name']}
+
+        ${helper.get_function_docstring(f, False, config, indent=8)}
+        '''
+        import sys
+
+        if num_samples is None and wfm is None:
+            raise TypeError("Either 'num_samples' or 'wfm' must be set")
```

Comment:
I vote: remove wfm parameter, make num_samples not optional.

Reply:
I vote: keep wfm parameter, num_samples optional.